### DISCUSSION TITLE
HA 2024.12.0 and failure of integration to load

### DISCUSSION THREAD

**Discussion by KirbyCub** [bigmoby/fglair_for_homeassistant#214] [General]
Since installing 2024.12.0, This has been showing up in notifications «Setup failed for custom integration 'fglair_heatpump_controller': Requirements for fglair_heatpump_controller not found: ['aiofiles==23.2.1'].»

Any suggestions? I was largely dependent on this integration to control my heat, and this comes at an unfortunate time.
Thanks ✌️😉

**Comment by emielverstegen** [comment]
Same issue here after upgrading to 2024.12.0. I see the requirements.txt has been updated already in the github repository, but no update available yet in HACS. 

Work around to get it working again for me: go to custom_components/fglair_heatpump_controller/manifest.json and change "aiofiles==23.2.1" to "aiofiles==24.1.0". Save file and restart home assistant.

**Reply by jarube** [reply]
yes, that did the trick for now. Thanks.

**Comment by bigmoby** [comment]
Hi guys, the latest version (https://github.com/bigmoby/fglair_for_homeassistant/releases/tag/v0.3.26) fixes the issue.
